packages/story-editor/src/components/hotlinkModal/useHotlinkModal.js

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { useCallback, useState } from '@googleforcreators/react';
+import { withProtocol } from '@googleforcreators/url';
+import { __ } from '@googleforcreators/i18n';
+/**
+ * Internal dependencies
+ */
+
+import useCORSProxy from '../../utils/useCORSProxy';
+import useAPI from '../../app/api/useAPI';
+import {
+  getHotlinkDescription,
+  isValidUrlForHotlinking,
+  getErrorMessage,
+} from './utils';
+
+function useHotlinkModal({ onSelect, onClose, onError, allowedFileTypes }) {
+  const [isInserting, setIsInserting] = useState(false);
+  const [link, setLink] = useState('');
+  const [errorMsg, setErrorMsg] = useState(false);
```

Comment:
```suggestion
  const [errorMsg, setErrorMsg] = useState(null);
```

This helps avoid type warnings (for example in PhpStorm but theoretically also in TypeScript) when later setting strings here when the initial value is a boolean.

I know this might have been different on `main`, but it's a good opportunity to change this.

Reply:
Fixed.